### PR TITLE
hcl to yaml in docs

### PR DESCRIPTION
**Pull request by sinan**
converts `hcl` code snippets to `yaml` in terraform docs.

img
img

see: https://sinan.github.io/koding/docs/terraform/providers/aws/index.html/

### CODE REVIEW

File: docs/_layouts/doc.html [hunk 1]
```diff
@@ -47,7 +45,76 @@
...
+            wasJSON = false,
+            block = el.parent().parent(),
+            tabs = "<nav><a href='#' class='yaml active'>yaml</a><a href='#' class='hcl'>hcl</a></nav>",
+            hcl = el.text();
+
+        try {
+
+          // dirty work around #0
+          // do nothing if single line
+          if (!/\n/.test(hcl)) throw 'nah!';
```
Comment (sinan) [inline docs/_layouts/doc.html]: img
not meant in turkish 😏

File: docs/_layouts/doc.html [hunk 2]
```diff
@@ -47,7 +45,76 @@
...
+          // dirty work around #2
+          // if starts with `$ ` or `export ` or `terraform `
+          if (/^\$|^export\s|^terraform\s/.test(hcl)) throw 'nah!';
+
+          // if it's already json
+          try {
+            obj = JSON.parse(hcl);
+            tabs = "<nav><a href='#' class='yaml active'>yaml</a><a href='#' class='json'>json</a></nav>";
+          } catch(e) {
+            obj = hcltojson(hcl);
```
Comment (gokmen) [inline docs/_layouts/doc.html]: it would be better to take this `hcltojson(hcl)` part in to a `try..catch` block.

**Review by gokmen — COMMENTED** [review]
LGTM just this minor in case of something goes wrong in converter.

**Review by gokmen — APPROVED** [review]

**Comment by gokmen** [comment]
💯

File: docs/_layouts/doc.html (hunk 2 above)
Comment (sinan) [inline docs/_layouts/doc.html]: i know it's a bit counterintuitive but the outer `try` is for that one.

**Pull request merged**